lib/helper/FileSystem.js

```diff
@@ -59,6 +59,14 @@ class FileSystem extends Helper {
     assert.ok(fileExists(this.file), `File ${name} not found in ${this.dir}`);
   }
 
+  /**
+  * Checks that any file exists including given text
```

Comment:
```suggestion
  * Checks that file with a name including given text exists in the current directory
```

Reply:
Please add an example with JavaScript code here. You can use the code from your test here.